_posts/2021-05-23-irc-move.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "NewPipe bides farewell to freenode and can now be found on Libera.Chat"
+short: "NewPipe moves to Libera.Chat"
+date: 2021-05-23 09:00:00 + 1:00
+categories: [pinned, release]
+author: Opusforlife & Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+We wrote an in-depth explanation below, but if you do not need or care about this, for you in a few words: Newpipe has an active IRC channel where developers and users chat and can ask questions. This channel moves from the freenode to the Libera.Chat network. Until Matrix finalizes its bridge to the new network, the room on freenode will be active, but as soon as that happens, it will be shut down.
+<!-- more -->
+
+## What is IRC?
+
+In order to provide a bit of a background, what follows will be short explanation what all this is. If you are familiar with IRC, skip to the next section.
+
+[freenode](https://freenode.net/) and [Libera.Chat](https://libera.chat) are IRC networks. IRC stands for Internet Relay Chat. Without diving too deep in technical explanations (if you care for this, read the [wikipedia article](https://en.wikipedia.org/wiki/Internet_Relay_Chat)), it is an instant communication standard for groups. You install an IRC app on your device (or use a webapp), those act as IRC clients, which you then need to point to an IRC server. Bigger IRC servers are called networks, because instead of just one server, they are a network of several servers (makes sense). The biggest and oldest IRC network is called freenode, and we as NewPipe used this network to host our channel. Up until this week.
```

Comment:
```suggestion
[freenode](https://freenode.net/) and [Libera.Chat](https://libera.chat) are IRC networks. IRC stands for Internet Relay Chat. Without diving too deep into technical explanations (if you care for this, read the [wikipedia article](https://en.wikipedia.org/wiki/Internet_Relay_Chat)), it is an instant communication standard for groups. You install an IRC client app on your device (or use a webapp), which you then need to point to an IRC server. Bigger IRC servers are called networks, because instead of just one server, they are a network of several servers (makes sense). The biggest and oldest IRC network is called freenode, and Team NewPipe used this network to host our channel. Until this week, this was perfectly fine and no one batted an eye regarding any of this.
```